magprime/templates/emails/attendee_swag_promo.html

```diff
@@ -0,0 +1,327 @@
+                    <tr>
+                        <td style="padding: 20px 0px 30px 0px;" class="logo">
+                            <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <td bgcolor="#333333" width="100" align="center">
+                                        <a href="http://magfest.org" target="_blank">
+                                            <img alt="MAGFest" src="http://magfest.org/images/2016swag/m2016titlelarge.png" width="300" height="80" style="display: block; font-family: Helvetica, Arial, sans-serif; color: #ffffff; font-size: 16px;" border="0">
+                                        </a>
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                </table>
+            </div>
+        </td>
+    </tr>
+</table>
+
+<!-- ONE COLUMN SECTION -->
+<table border="0" cellpadding="0" cellspacing="0" width="100%">
+    <tr>
+        <td bgcolor="#ffffff" align="center" style="padding: 20px 15px 70px 15px;" class="section-padding">
+            <table border="0" cellpadding="0" cellspacing="0" width="500" class="responsive-table">
+                <tr>
+                    <td>
+                                    <!-- COPY -->
+                                    <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                                        <tr>
+                                            <td align="center" style="font-size: 25px; font-family: Helvetica, Arial, sans-serif; color: #333333; padding-top: 0px;" class="padding-copy">MAG-Swag</td>
+                                        </tr>
+                                        <tr>
+                                            <td align="left" style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;" class="padding-copy">
+                                              
+                                              <p>
+                                                Hey there {{ attendee.first_name}}!
+                                              </p>
+<p>
+Thank you so much for registering to attend MAGFest 2016!<br>
+When you registered, you had the option to also pre-order extra swag items and badge upgrades, most of which are offered exclusively during pre-registration. You can check out what swag you've already pre-ordered, and if you'd like, upgrade your swag tier, using this
+    <a href="{{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }}">personalized registration update link</a>.
+</p>
+<p>
+Make your choices soon, though, because the last day to order any item at the key chain tier or above is <strong>December 31st!</strong>
+</p>
+<p>
+Here's a preview of some of the cool stuff you can get if you order before the deadline:
```

Comment:
Should we make these depend on whether they've already gotten them?  Like, if someone has already bought a tshirt, maybe there's no need to advertise the tshirt.